relay-metrics/src/aggregation.rs

```diff
@@ -846,13 +846,18 @@ impl Ord for QueuedBucket {
 ///   failed buckets. They will be merged back into the aggregator and flushed at a later time.
 #[derive(Clone, Debug)]
 pub struct FlushBuckets {
+    /// the project key
+    pub public_key: ProjectKey,
```

Comment:
Let's not make this public and rather add an accessor method.

Reply:
ok